car.pde

```diff
@@ -107,40 +149,32 @@ boolean collides(Collideable a, Collideable b){
   return true;
 }
 
-boolean inside(Collideable a, Collideable b){

```

Comment:
Funkcija inside iako nije korištena radi i ima smisla s obzirom na trenutni način na koji igra funkcionira.

Reply:
Ali nicemu ne sluzi?